### ISSUE TITLE
Disclaimer of Graion Dilach's.

### ISSUE THREAD

**Issue by GraionDilach** [Phobos-developers/Phobos#1081]
### Description

This is a disclaimer of Graion Dilach, aka Gyula Zimmermann.

I am hereby granting the agreement of relicensing all my commits made within the repositories contained in https://github.com/Ares-Developers to the WTFPL (and every compatible license, which includes the public domain in areas where WTFPL is not acknowledged) or any other license compatible with Phobos, depending on your local rules or jurisdiction. In the scenarios/areas where there isn't a Phobos-compatible relicensing public domain license available, I am willing to relicense the above contributions (direct or indirect) to the license currently employed by this project.

This is a non-rectactable, legally binding statement, granted in a public ticket for transparency and reference material.

Thank you.

### Phobos Version

N/A

### Conditions to reproduce

N/A

### INI code

```ini
N/A
```

### Steps to reproduce

N/A

### Expected behaviour

N/A

### Actual behaviour

N/A

### Additional context

N/A

### Checklist

- [X] The issue happens on the **latest official** version of Phobos and wasn't fixed yet.
- [X] I agree to elaborate the details if requested and provide thorough testing if the bugfix is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, gifs, videos, debug and exception logs, etc)*.

**Comment by Metadorius** [comment]
As one of the project founders I want to express my (and our collectively) gratitude for this decision, especially considering the circumstances under which the project was born and the fact that currently Phobos de-facto is a successor of Ares.

Thank you!

**Comment by GraionDilach** [comment]
Yeah, sorry for abusing the bugtracker for this, but I wanted to have a written statement publicly available somewhere.

**Comment by Metadorius** [comment]
It's absolutely not a problem, but we can convert it (or just repost it there) to a discussion (and remove the unused fields, the edit history) if you want.

**Comment by GraionDilach** [comment]
Honestly, I don't trust GH discussions for this. They've already changed formatting twice, despite not being a lot younger feature. Providing the statement in a ticket sounds more futureproof to me.

**Comment by wardeathfun** [comment]
Not trying to be ungrateful, but wtf did you ever do? Is Ares open source now as it should have been from the start when you hijacked the original license or what? I only see your name in ppm dramas and nothing more. The big goodbye from a PIFF who bent the knee before latinos such ass Banshee who uses Adamstrange and G-E as 2nd accounts to keep his trash alive. Big applause nevertheless. Just burn already you Victor Orban pig.

**Comment by GraionDilach** [comment]
Ares was never open-source. The 4-clause BSD license it uses isn't an open-source license because it includes the advertising claim. Additionally, all contributors need to agree to a possible relicensing. I've made this statement because I've retired from the C&C community, there aren't many people I still keep contact with from it and I wanted to prevent any opportunity incase such a relicensing option ends up blocked on me.

While I didn't do much, my permission is required due to my contributions to Ares AttachEffect, Abductor.ChangeOwner and some preparations for HouseColorBolts. More exactly; these: https://github.com/Ares-Developers/Ares/commits?author=GraionDilach.

This statement allows me to emjoy my time in a community which welcomes and heavily appreciates my contributions to it while also allows any Phobos-affiliate to mediate with the other Ares developers in my name and/or consent. It's a blanket statement, sure, but these are the blanket statements required for such an option.
